community/crystal/APKBUILD

```diff
@@ -1,9 +1,9 @@
 # Contributor: Jakub Jirutka <jakub@jirutka.cz>
 # Maintainer: Jakub Jirutka <jakub@jirutka.cz>
 pkgname=crystal
-pkgver=0.27.0
+pkgver=0.27.2
 pkgrel=0
-_bootver=0.26.0
+_bootver=0.27.0
 _llvmver=5
```

Comment:
I think you should be able to use llvm 6. There are some issues with 7, though.

Reply:
is llvm 6 available for alpine? couldn't see it here: https://pkgs.alpinelinux.org/packages?name=llvm6*&branch=edge

Reply:
Oh, I simply assumed it was. Nevermind then.